_overviews/compiler-options/errors.md

```diff
@@ -0,0 +1,103 @@
+# Introduction
+
+An advanced mechanism for formatting type errors and inspecting missing
+implicits has been introduced in Scala 2.13.6.
+It is based on the compiler plugin [splain](https://github.com/tek/splain).
+
+This tool abstracts several classes of compiler errors with simple data types
+that can be processed by a few built-in routines as well as
+[user-provided analyzer plugins](/overviews/plugins/index.html).
+
+The most significant feature is the illustration of chains of implicit instances
+that allows a user to determine the root cause of an implicit error:
+
+![implicits](/resources/img/implicits-circe.jpg)
+
+# Basic Configuration
+
+* `-Vimplicits` enables printing of implicit chains
+* `-Vtype-diffs` enables colored diffs for found/required errors
+
+## Additional Configuration
+
+`-Vimplicits-verbose-tree` shows the implicits between the error site and the
+root cause, see [#implicit-resolution-chains].
+
+`-Vimplicits-max-refined` reduces the verbosity of refined types, see
+[#truncating-refined-types].
+
+# Features
+
+The error formatting engine provides the following enhancements:
+
+## Infix Types
+
+Instead of `shapeless.::[A, HNil]`, prints `A :: HNil`.
+
+## Found/Required Types
+
+Rather than printing up to four types, only the dealiased types are shown as a colored diff:
+
+![foundreq](/resources/img/foundreq.jpg)
+
+## Implicit Resolution Chains
+
+When an implicit is not found, only the outermost error at the invocation point is printed by the regular error
+reporter.
+Previously, the flag `-Xlog-implicits` caused the compiler to print all information about processed implicits, but the
+output was highly verbose and contained all invalid implicits for parameters that have been resolved successfully.
+The flag is now an alias for `-Vimplicits`, which prints a compact list of all involved implicit instances:
```

Comment:
👨🏼‍🎨  Very pedantically, I'd say it's renamed to -Vimplicits and the old -Xlog-implicits will continue to work as a deprecated alias.  Minor itch :)

Reply:
please don't hold back on the pedantry!